src/signTxPoolRegistration.c

```diff
@@ -750,18 +1333,46 @@ static void signTxPoolRegistration_handlePoolMetadataAPDU(uint8_t* wireDataBuffe
 // ============================== CONFIRM ==============================
 
 enum {
-	HANDLE_CONFIRM_STEP_FINAL_CONFIRM = 6360,
+	HANDLE_CONFIRM_STEP_FINAL_NO_OWNERS = 7100,
+	HANDLE_CONFIRM_STEP_FINAL_NO_RELAYS,
+	HANDLE_CONFIRM_STEP_FINAL_CONFIRM,
 	HANDLE_CONFIRM_STEP_RESPOND,
 	HANDLE_CONFIRM_STEP_INVALID,
 };
 
 static void signTxPoolRegistration_handleConfirm_ui_runStep()
 {
 	TRACE("UI step %d", subctx->ui_step);
+	TRACE_STACK_USAGE();
 	ui_callback_fn_t* this_fn = signTxPoolRegistration_handleConfirm_ui_runStep;
 
-	UI_STEP_BEGIN(subctx->ui_step);
-
+	UI_STEP_BEGIN(subctx->ui_step, this_fn);
+
+	// we display potencially suspicious facts about the certificate
+	// that have not been explicitly shown to the user before:
+	// missing owners or relays
+	UI_STEP(HANDLE_CONFIRM_STEP_FINAL_NO_OWNERS) {
+		if (subctx->numOwners == 0) {
+			ui_displayPaginatedText(
+			        "No pool owners",
+			        "",
+			        this_fn
+			);
+		} else {
+			UI_STEP_JUMP(HANDLE_CONFIRM_STEP_FINAL_NO_RELAYS);
+		}
+	}
+	UI_STEP(HANDLE_CONFIRM_STEP_FINAL_NO_RELAYS) {
+		if (subctx->numRelays == 0) {
```

Comment:
showing this in "owner mode" is confusing as we don't show the relays at all, if present